skeletons/src/dsHdmiIn.c

```diff
@@ -200,12 +200,3 @@ dsError_t dsSetEdid2AllmSupport(dsHdmiInPort_t iHdmiPort, bool allmSupport)
   (void)allmSupport;
   return (dsError_t)0;
 }
-
-dsError_t dsGetEdid2AllmSupport(dsHdmiInPort_t iHdmiPort, bool *allmSupport)
-{
```

Comment:
Please consider to change in the test spec documents as well - ds-hdmi-in-L2-Low-Level_TestSpec.md and ds-hdmi-in-High-Level_TestSpec.md

Reply:
done